### PR TITLE
Manage Products V2

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
Rob (Krentz) designed changes to the manage product screen as part of the designs for the new Digi Sub flow. This PR creates a new version of the manage products screen (due to the complexity of untangling the existing file and the decision to not roll the previous refactor out to all products)

Designs can be found [here](https://www.figma.com/file/sx9jlOcpcK9V0gNgj4GWtW/Cancellation-saves-Q3-2023?type=design&node-id=155-52569&mode=dev)

**This screen is not linked up yet so is only viewed via stroybook**

## How to test
View new screen on storybook

## Images
image

image## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

**Comment by charleycampbell** [comment]
@charleycampbell to check spacing and mobile view

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+			{shouldContactUsToCancel && (
+				<div
+					css={css`
+						${textSans.medium()};
+						color: ${palette.neutral[46]};
+						margin-top: 16px;
+						justify-content: center;
+					`}
+				>
+					Would you like to cancel your {props.friendlyName}?
```
Comment (andrewHEguardian) [inline client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx:294]: in the figma it looks like it just says "Cancel subscription"

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+	const groupedProductType =
+		GROUPED_PRODUCT_TYPES[specificProductType.groupedProductType];
+
+	const hasCancellationPending = productDetail.subscription.cancelledAt;
+
+	const cancelledCopy =
+		specificProductType.cancelledCopy || groupedProductType.cancelledCopy;
+
+	console.log(nextPaymentDetails?.paymentInterval);
```
Comment (andrewHEguardian) [inline client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx:96]:
```suggestion
```

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+
+	const monthlyOrAnnual = calculateMonthlyOrAnnualFromBillingPeriod(
```
Comment (andrewHEguardian) [inline client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx:98]: I think a digisub can be paid quarterly

**Review by andrewHEguardian — APPROVED** [review]
Noticed a couple of small things but looks good! 🎨

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import { digitalPackPaidByDirectDebit } from '../../../../fixtures/productBuilder/testProducts';
+
+export default {
+	title: 'Pages/ManageProductV2',
+	component: ManageProductV2,
+	decorators: [ReactRouterDecorator],
+	parameters: {
+		reactRouter: {
+			state: { productDetail: digitalPackPaidByDirectDebit() },
+			nextPaymentDate: '20 March',
```
Comment (andrewHEguardian) [inline client/components/mma/accountoverview/manageProducts/ManageProductV2.stories.tsx:14]: does this get used?

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+					.
+				</p>
+			)}
+
+			<BasicProductInfoTable
+				groupedProductType={groupedProductType}
+				productDetail={productDetail}
+			/>
+
+			<h2
```
Comment (andrewHEguardian) [inline client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx:153]: probably should be an `h3` if the Manage bit is an `h2`

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+				css={css`
+					${textSans.medium({ fontWeight: 'bold' })}
+					font-style: normal;
+					font-size: 20px;
+					font-weight: bold;
```
Comment (andrewHEguardian) [inline client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx:158]: this is redundant and should be covered by the `textSans` styles

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx (hunk 1 above)
Comment (charleycampbell) [reply client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx:288]: This is temporary until the self cancellation flag is set to true...we could change it at the moment but I would prefer to keep it as is so we remember to do it!

**Review by andrewHEguardian — APPROVED** [review]
Looks good 👍 🚀

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx (hunk 5 above)
Comment (charleycampbell) [reply client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx]: great spot

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.stories.tsx (hunk 4 above)
Comment (charleycampbell) [reply client/components/mma/accountoverview/manageProducts/ManageProductV2.stories.tsx]: nah I've removed it!

File: client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx (hunk 3 above)
Comment (charleycampbell) [reply client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx]: Refactored this to include quarterly

**Pull request merged**